src/components/victory-bar.jsx

```diff
@@ -358,72 +369,96 @@ export default class VictoryBar extends React.Component {
     if (axis !== "x" || !props.categories || Util.Collection.containsStrings(props.categories)) {
       return undefined;
     }
-    return [_.min(_.flatten(props.categories)), _.max(_.flatten(props.categories))];
+    return [Math.min(..._.flatten(props.categories)), Math.max(..._.flatten(props.categories))];
```

Comment:
grabs all of the values out of the resulting array. `Math.min` expects a list of args instead of an array. Does look pretty ugly though